src/containers/Stewardship/index.tsx

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import DefaultContent from './StewardshipContent';
-import commonUtils from '../../lib/commonUtils';
+//import React from 'react';
```

Comment:
@kathleenahall This code here should remain uncommented out as it will never be visible